internal/dataplane/kongstate/util_test.go

```diff
@@ -142,6 +143,38 @@ func TestKongPluginFromK8SClusterPlugin(t *testing.T) {
 			want:    kong.Plugin{},
 			wantErr: true,
 		},
+		{
+			name: "Config and ConfigPatches set",
```

Comment:
Can we please add test cases for error conditions? That would be for example:
- missing key in the Secret
- invalid `Patch.Path` 
- invalid value in the Secret's key (e.g. non-quoted string).

Reply:
done.